MiniDumper/MiniDumper.cs

```diff
@@ -178,6 +180,28 @@ public void PrintDebugString(OUTPUT_DEBUG_STRING_INFO outputDbgStrInfo)
             }
         }
 
+        public void MemoryCommitThreshold(int commitThreshold)
+        {
+            System.Timers.Timer aTimer = new System.Timers.Timer();
+            aTimer.Interval = 1000;
```

Comment:
I would like to have the interval configurable (as in procdump) using the **-s** option.

Reply:
this interval for pooling interval. in procdump also they used 1000 ms. they said in Defrag tools.

-s | Consecutive seconds before dump is written (default is 10)
i think this is different.

Reply:
Ach yes, you are right. This interval just configures the minimum amount of time between subsequent dumps (when of course more than one dumps was requested). So we do not dump on every second.